src/uint/bits.rs

```diff
@@ -1,9 +1,20 @@
 use crate::{Limb, UInt, Word};
 
 impl<const LIMBS: usize> UInt<LIMBS> {
+    /// Get the value of the bit at position `index`, as a 0- or 1-valued Word.
+    /// Returns 0 for indices out of range.
+    #[inline(always)]
+    pub const fn bit_vartime(self, index: usize) -> Word {
+        if index >= LIMBS * Limb::BIT_SIZE {
+            0
+        } else {
+            (self.limbs[index / Limb::BIT_SIZE].0 >> (index % Limb::BIT_SIZE)) & 1
+        }
+    }
+
     /// Calculate the number of bits needed to represent this number.
     #[allow(trivial_numeric_casts)]
-    pub const fn bits(self) -> usize {
```

Comment:
Since `master` is still v0.4.x, can you keep this around with a `#[deprecated]` attribute, calling `bits_vartime`?

We can remove it in the next breaking release.